test/factory/CloneFactory.ts

```diff
@@ -23,10 +25,8 @@ describe('CloneFactory', function () {
 
           await instance['deployClone()']();
 
-          expect(
-            await ethers.provider.getCode(address)
-          ).to.equal(
-            await ethers.provider.getCode(instance.address)
+          expect(await ethers.provider.getCode(address)).to.equal(
```

Comment:
This format is painful to read.  The work of `prettier`, I assume?

Reply:
Yes, prettier.

Reply:
Has prettier formatting been applied to all files?  Some are worse than others.

Reply:
Yes it has